### ISSUE TITLE
FDMS Glossary Webpage  - Review

### ISSUE THREAD

**Issue by cassandrajHall** [GSA-TTS/OROS-eRulemaking#28]

**Comment by gsawdshaw** [comment]
I agree we can scuttle this feature for now and pick it up as part of modernization, or even stabilization. Context-sensitive help?

**Comment by ianhabit-gsa** [comment]
I'm cool with deprecating this as well, wish we had usage metrics to confirm.

**Comment by cassandrajHall** [comment]
Adding FDMS glossary usage for your review.
![image (2)](https://github.com/GSA-TTS/OROS-eRulemaking/assets/169190035/23685868-90f9-4651-b85b-b2dedbe85f05)
![image (1)](https://github.com/GSA-TTS/OROS-eRulemaking/assets/169190035/4e8430d4-f534-42cb-8364-55c9a1f2140d)
